app/templates/releases/index.hbs

```diff
@@ -25,7 +25,29 @@
 
 <h2>Upcoming releases</h2>
 
-<ReleaseTimeline class="hide-on-mobile" @project={{emberBetaProject}} />
+The Ember project aims to release approximately every 6 weeks.
+For a very rough estimate of the timeline for the next release,
+you can look at the date of the most recent
+<a href="https://blog.emberjs.com/tags/releases.html">
+  official release blog post
+</a>
+and count forward by six weeks.
```

Comment:
This is not accurate. The six week schedule is independent of delays. For a practical example, 3.19 was "released" this week and 3.20 is still scheduled for the week of July 6, two weeks from now.